docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1649,53 +1649,85 @@ fits the encoded integer value. This representation achieves three goals:
 
 * It helps facilitate non-disruptive P4 program updates.
 
-The shortest possible binary string for an integer `value` with `high_bit`
-as the most-significant non-zero bit in the value is computed as:
+In particular, the kinds of P4 program updates that this
+representation facilitates are those where a P4Runtime server and
+client can continue to transmit P4Runtime messages between them when
+one has a P4Info file for version A of a P4 program, at the same time
+that the other has a P4Info file for version B of a P4 program, and
+those P4 programs differ in the bit widths of some values of type
```

Comment:
I think the rest of the spec uses the "bitwidth" syntax

Reply:
Done in the new branch I created in this repo: https://github.com/p4lang/p4runtime/tree/issue111-change-signed-int-representation